src/chrome/content/rules/Jimdo-Server.com.xml

```diff
@@ -0,0 +1,28 @@
+
+<!--
+	Other Jimdo rulesets:
+		-Jimdo.xml
+		-Jimcdn.com.xml
+		-Jimstatic.com.xml
+		-Jimdo.design.xml
+
+	Nonfunctional subdomains:
+		jimdo-server.com
+		
+-->
+<ruleset name="Jimdo-Server.com (partial)">
+
+	<!--	Direct rewrites:
+				-->
+				
+	<target host="mgmt.jimdo-server.com" />
+	<target host="api.dmp.jimdo-server.com" />
+	<target host="devkit.dmp.jimdo-server.com" />
+	<target host="screenshots.dmp.jimdo-server.com" />
```

Comment:
Do you have a working test url for this?

Reply:
Also please sort the targets alphabetically

Reply:
Do you want a test in this ruleset for this domains? Or do you want to see where this targets used?

Reply:
A test in this ruleset for this domain

Reply:
`&` does not work with our test. You can replace it with `&amp;`